openpyxl-stubs/workbook/workbook.pyi

```diff
@@ -43,11 +43,13 @@ class Workbook:
         scope: Optional[int] = ...,
     ) -> None: ...
     def create_sheet(
-        self, title: Optional[Union[str, int]] = ..., index: None = ...
+        self, title: Optional[Union[str, int]] = ..., index: Union[None, int] = ...
```

Comment:
Not strictly related, happy to split this out into a separate PR if you prefer https://openpyxl.readthedocs.io/en/stable/api/openpyxl.workbook.workbook.html?highlight=workbook#openpyxl.workbook.workbook.Workbook.create_sheet